### PR TITLE
Regenerate users.yaml

### PR DESCRIPTION
**Pull request by jturel**

### CODE REVIEW

**Review by ekohl — CHANGES_REQUESTED** [review]
Do we really need to clean up old entries? If so, there's a lot more that. I also think https://github.com/theforeman/prprocessor/blob/master/scripts/generate_user_list.rb will add them back again unless we remove them from the orgs as well.

**Comment by jturel** [comment]
> Do we really need to clean up old entries? If so, there's a lot more that. I also think https://github.com/theforeman/prprocessor/blob/master/scripts/generate_user_list.rb will add them back again unless we remove them from the orgs as well.

I don't see any point in keeping them. Thanks for letting me know about the script - I used that instead of manual edits.

File: config/users.yaml [hunk 1]
```diff
@@ -143,3 +143,10 @@ hao-yu: '15770'
 wbclark: '16866'
 domitea: '20759'
 lstejska: '23176'
+grizzthedj: ''
```
Comment (ekohl) [inline config/users.yaml:146]: 17420

File: config/users.yaml [hunk 2]
```diff
@@ -143,3 +143,10 @@ hao-yu: '15770'
...
+isimluk: ''
```
Comment (ekohl) [inline config/users.yaml:147]: 4689

File: config/users.yaml [hunk 3]
```diff
@@ -143,3 +143,10 @@ hao-yu: '15770'
...
+laugmanuel: ''
```
Comment (ekohl) [inline config/users.yaml:148]: 23996

File: config/users.yaml [hunk 4]
```diff
@@ -143,3 +143,10 @@ hao-yu: '15770'
...
+Odilhao: ''
```
Comment (ekohl) [inline config/users.yaml:149]: 48173

File: config/users.yaml [hunk 5]
```diff
@@ -143,3 +143,10 @@ hao-yu: '15770'
...
+rabajaj0509: ''
```
Comment (ekohl) [inline config/users.yaml:150]: 8740

File: config/users.yaml [hunk 6]
```diff
@@ -143,3 +143,10 @@ hao-yu: '15770'
...
+lfu: ''
```
Comment (ekohl) [inline config/users.yaml:151]: 44796

File: config/users.yaml [hunk 7]
```diff
@@ -143,3 +143,10 @@ hao-yu: '15770'
...
+rverdile: ''
```
Comment (ekohl) [inline config/users.yaml:152]: 47478

**Review by ekohl — CHANGES_REQUESTED** [review]

**Comment by jturel** [comment]
Lol, didn't even notice the script didn't populate those values. Seems half-baked

**Comment by jturel** [comment]
Updated ;)

**Comment by ekohl** [comment]
> Lol, didn't even notice the script didn't populate those values. Seems half-baked

The git log for https://github.com/theforeman/prprocessor/blob/master/scripts/generate_user_list.rb indicates it was last modified 6 years ago. Clearly we haven't had enough need for it to implement it ;)

**Review by ekohl — APPROVED** [review]

**Pull request merged**